roles/transformers/tasks/main.yml

```diff
@@ -93,22 +70,24 @@
   register: job_result
   until: job_result.finished
   delay: 30
-  retries: 30
-  when: loffice_exists.rc != 0
+  retries: 20
 
 - name: Extract libreoffice-dist-linux.gz
   become: true
   unarchive:
     src: "{{ download_location }}/libreoffice.tar.gz"
     dest: "{{ download_location }}/"
     remote_src: true
-  when: loffice_exists.rc != 0
+    extra_opts:
```

Comment:
I don't really know what this is :(

Reply:
This makes sure the extracted archive have a predictable root folder name.
Doing so, we can:
* have the archive extracted to the right place straight away
* avoid having an extract to temp, copy to dest, remove temp set of tasks
* use the "creates" argument of the unarchive module
* avoid using registered test tasks and when conditions

Reply:
I knew about "creates" bit. The extra_opts confused me :). Thank you